pulp_smash/utils.py

```diff
@@ -9,6 +9,7 @@
 from packaging.version import Version
 from pulp_smash.constants import REPOSITORY_PATH, USER_PATH
 from time import sleep
+from urlparse import urljoin
```

Comment:
> Do you prefer that I undo this use of urljoin and file an issue to change them all at once?

I'd like to do this all at once through the entire `pulp_smash.utils` module. Having more targeted commits makes it easier to do things like use `git bisect`, revert commits and walk through the commit log. It's getting toward the end of the day - I'd be delighted if we get a GitHub issue up by the end of today.